build/config/SignToolData.json

```diff
@@ -111,6 +111,7 @@
         "Microsoft.CodeAnalysis.Elfie.dll",
         "Microsoft.DiaSymReader.dll",
         "Microsoft.DiaSymReader.Native.amd64.dll",
+        "Microsoft.DiaSymReader.Native.arm.dll",
```

Comment:
This is a fantastic question for which I have no answer. The project.lock.json is stonewalling me.